frontend/src/components/GameComponent/gameparts/About.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react'
+import rdr2 from '../../asseets/rdr2.jpg'
+import {FaWindows, FaPlaystation, FaXbox} from 'react-icons/fa'
+
+
+function About() {
+  return (
+    <div>
+        <div className='outermost-div'>
+        <div className='prices-outer-div'>
+           <div className='iframe-container'> 
+        <iframe className='responsive-iframe' src="https://www.youtube.com/embed/QkkoHAzjnUs"  frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+        </div>
+
+        <div className='some-detail-res'>
+      
+    
+    <div className='inner-div'>
+    <div>
+      <p>Release Date:<br></br> 02 March 2023</p>
+    </div>
+    <div>
+      <p>Platforms:<br/>
+        <FaWindows style={{marginRight:'0.5rem'}}/><FaPlaystation  style={{marginRight:'0.5rem'}}/><FaXbox style={{marginRight:'0.5rem'}}/>
+      </p>
+
+
+    </div>
+
+    <div>
+      Developer / Publisher: <br/>
+      <h4 style={{textDecoration:'underLine'}}>Arpit / Gulati</h4>
+    </div>
+    <div>Genres:<br/>
+    <div style={{display:'flex', flexWrap:'wrap'}}>
+    <h5>Simulation</h5>
+    <h5>RolePlay</h5>
+    <h5>Fictional World</h5>
+    </div>
+    </div>
+
+    <div>Tags:<br/>
+    <div style={{display:'flex', flexWrap:'wrap'}}>
+    <h5 >Adventure</h5>
```

Comment:
It's better to create an array and add all values in array, and use map instead of writing multiple times.